File: RemoveDuplicatesFromSortedArray.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution 
{
    public:
        int removeDuplicates(vector<int>& nums) 
        {
            int cur=0;
            int n=nums.size();
            for(int i=0;i<n;i++)
            {
                if(i==n-1 || nums[i]!=nums[i+1])
                {
                    nums[cur++]=nums[i];
                }
                
            }
            return cur;
        }
};
int main()
{
    
    return 0;
}

File: RemoveDuplicatesFromSortedArray.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let i=0;
    while(i<nums.length)
    {
        j=i+1;
        while(j<nums.length && nums[j]==nums[i])
        {
            j++;
        }
        if(j!=i+1)
        {
            nums.splice(i+1,j-(i+1));
        }
        i++;
    }
};